packages/components/src/components/link/_link.scss

```diff
@@ -60,7 +60,16 @@
     color: $visited-link;
   }
 
-  .#{$prefix}--link--disabled {
+  #{$prefix}--link--disabled,
+  #{$prefix}--link--disabled:focus,
+  #{$prefix}--link--disabled:hover,
+  #{$prefix}--link--disabled:active,
+  #{$prefix}--link--disabled:visited,
+  .#{$prefix}--link.#{$prefix}--link--disabled,
+  .#{$prefix}--link.#{$prefix}--link--disabled:focus,
+  .#{$prefix}--link.#{$prefix}--link--disabled:hover,
+  .#{$prefix}--link.#{$prefix}--link--disabled:active,
+  .#{$prefix}--link.#{$prefix}--link--disabled:visited {
```

Comment:
Do we NEED both classes for the added weight or can we just have `bx--link--disabled` after the initial `bx--link` declarations?  We've ran in issues trying to overwrite selectors that have heavy weights so keeping them as low as possible would be ideal.